lib/ao_host.js

```diff
@@ -501,13 +494,8 @@ class AOHost extends AsyncEventEmitter {
 
     await this.maybeCancelExistingOrders(state, gid)
 
-    if (_isFunction(gidCB)) {
-      await gidCB(gid)
-    }
-
-    await this.emit('ao:start', this.instances[gid])
-
-    return gid
+    const data = await this._startAlgo(this.instances[gid])
```

Comment:
similar operation for stopAO missing on line: 
https://github.com/bitfinexcom/bfx-hf-algo/blob/7bdc1e5af2145619625979b9931d94c0f79d3af9/lib/ao_host.js#L543

```
const data = await this._stopAlgo(this.instances[gid])
return data
```

Reply:
good catch! fixing

Reply:
@robertkowalski this suddenly popped in my head haha.. this will be a problem

 https://github.com/bitfinexcom/bfx-hf-algo/blob/2321d1faa89b663a2611e3a7ed59c49ce771761e/lib/host/events/stop.js#L29